c2corg_common/fields_outing.py

```diff
@@ -12,7 +12,6 @@
     'locales.hut_comment',
     'locales.route_description',
     'geometry.geom',
-    'geometry.geom_detail',
```

Comment:
Do you really need those changes? If we don't need to add the attribute here, it's simpler to simply cancel this PR :P

Reply:
By the way, the version of v6_common to use (actually the hash of the last commit) is set here in the API: https://github.com/c2corg/v6_api/blob/master/requirements.txt#L51